lib/restivus.coffee

```diff
@@ -309,11 +311,19 @@ class @Restivus
       get: ->
         # Remove the given auth token from the user's account
         authToken = @request.headers['x-auth-token']
-        Meteor.users.update @user._id, {$pull: {'services.resume.loginTokens': {token: authToken}}}
+        hashedToken = Accounts._hashLoginToken authToken
+        tokenPaths = self.config.auth.token.split '.'
+        tokenPrefix = tokenPaths[..-2].join '.'
+        tokenSuffix = tokenPaths[-1..][0]
+        pullQueryTarget = {}
```

Comment:
I wasn't 100% excited about this area of the code.
I didn't find a good way to put the query together as needed.